### DISCUSSION TITLE
Is there a way to get error response?

### DISCUSSION THREAD

**Discussion by caralv27** [kean/Get#59] [General]
Is there a way to get the error response 
 For example if we are using the github api 
 The end point is 
`https://api.github.com/users/caralv27s`
And the error response is the following 
`{
"message": "Not Found",
"documentation_url": "https://docs.github.com/rest/reference/users#get-a-user"
}`
how can I grab that error response?

**Comment by kean** [comment]
Yes, you can use delegate's [client(_:validateResponse:data:task:)](https://kean-docs.github.io/get/documentation/get/apiclientdelegate/client(_:validateresponse:data:task:)-23iyu) to parse an error and throw instead of the default one.

**Reply by caralv27** [reply]
Thank you it works!
